proposals/lip-0065.md

```diff
@@ -187,12 +189,12 @@ The event `topics` is an array of unique byte values whose length ranges between
 
 By default, the first topic is set to:
 
-* the transaction ID, for events emitted during the transaction processing stages,
-* the cross-chain message ID, for events emitted during the cross-chain message processing stages,
 * the constant `EVENT_TOPIC_INIT_GENESIS_STATE`, for events emitted during the 'genesis state initialization' stage,
 * the constant `EVENT_TOPIC_FINALIZE_GENESIS_STATE`, for events emitted during the 'genesis state finalization' stage,
 * the constant `EVENT_TOPIC_BEFORE_TRANSACTIONS`, for events emitted during the 'before transactions execution' stage,
-* or the constant `EVENT_TOPIC_AFTER_TRANSACTIONS`, for events emitted during the 'after transactions execution' stage.
+* the constant `EVENT_TOPIC_AFTER_TRANSACTIONS`, for events emitted during the 'after transactions execution' stage,
+* the constant `EVENT_TOPIC_TRANSACTION_EXECUTION` concatenated with the transaction ID, for events emitted during the transaction processing stages,
+* the constant `EVENT_TOPIC_CCM_EXECUTION` concatenated with the cross-chain message ID, for events emitted during the cross-chain message processing stages.
```

Comment:
Here we should be careful on what do we define to be "ccm processing stages". By reading the LIP definitions, it seems to be the part where `ctx.ccmProcessing = True`. However, in LIP 53  we set `eventsDefaultTopic = ccmID` even [during `beforeCrossChainMessagesExecution`](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0053.md#beforecrosschainmessagesexecution). 

Ideally, the "ccm processing" should be well-defined such that we would not even need to set `eventsDefaultTopic` in LIP 53 and everything should follow from the specs defined here, in the same way that works for the other default topics.

Reply:
You are right, it is not super clear. `ctx.ccmProcessing = True` rather refers to the fee handling AND the event default topic, but since there are no fees handled in `[beforeCrossChainMessagesExecution](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0053.md#beforecrosschainmessagesexecution)`, we do not write it there. Any suggestion on how to improve this?

Reply:
Another option is to explicitly refer to LIP 53 and the line where we set the default topic

Reply:
Yep, perhaps just say that this can be identified by setting `eventsDefaultTopic = ccmID` during execution and add a link.